crates/holochain/src/core/workflow/sys_validation_workflow.rs

```diff
@@ -215,10 +253,17 @@ pub async fn sys_validation_workflow(
         trigger_self.trigger(&"sys_validation_workflow");
     }
 
+    if outcome_summary.warrant_deps_copied > 0 {
+        // If any warrant dependencies were copied, the workflow could attempt validation for those
+        // right away, so trigger it again.
+        trigger_self.trigger(&"sys_validation_workflow");
+    }
```

Comment:
Keep track of whether any warranted deps were copied from the cache to the DHT because that should immediately re-trigger the workflow to validate the newly inserted dependency